lib/cosmos/tools/cmd_tlm_server/cmd_tlm_server_config.rb

```diff
@@ -174,6 +174,15 @@ def process_file(filename, recursive = false)
                 raise parser.error("Unknown target #{target_name} mapped to interface #{current_interface_or_router.name}")
               end
 
+            when 'ADAPTER'
+              parser.verify_num_parameters(1, 1, "#{keyword} <adapter.rb>")
```

Comment:
Adapters need to support parameters or they have limited use.  CRC settings for example.  Should take any given optional parameters and call some method on the interface (set_adapter_settings) which stores them away in a hash indexed by adapter filename.